templates/components/common/navigation-list.html

```diff
@@ -1,26 +1,25 @@
 {{#if children}}
     <a 
-        class="js-nav-pages-action has-sub-menu{{#if is_active}} is-active{{/if}}"
+        class="js-nav-pages-action has-sub-menu{{#if is_active}} is-active{{/if}}{{#if active_category_navigation '===' 'true'}} is-active{{/if}}"
         href="{{url}}"
         data-collapsible="nav-pages-{{id}}"
     >
         {{name}}
     
-        <svg>
-            <use href="#icon-chevron-down" />
+        <svg xmlns="http://www.w3.org/2000/svg" width="25" height="24" viewBox="0 0 25 24" fill="none">
+            <mask id="mask0_2973_30873" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="25" height="24">
+                <path d="M6.5 9L12.5 15L18.5 9" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+            </mask>
+            <g mask="url(#mask0_2973_30873)">
+                <rect x="0.5" width="24" height="24" fill="{{#if is_active}}#5A64C1{{else}}#272727{{/if}}"/>
+            </g>
         </svg>
     </a>
-    <div id="nav-pages-{{id}}" aria-hidden="true" tabindex="-1">
-        <ul>
-            <li>
-                <a 
-                    class="js-nav-pages-action"
-                    href="{{url}}"
-                    aria-label="{{lang 'category.view_all.name' category=name}}"
-                >
-                    {{lang 'category.view_all.name' category=name}}
-                </a>
-            </li>
+    <div class="c-navigation__megamenu" id="nav-pages-{{id}}" aria-hidden="true" tabindex="-1">
```

Comment:
File is called `navigation-list` so classnames should start with `.c-navigation-list` :)